project/src/main/java/com/google/sps/data/EventResult.java

```diff
@@ -0,0 +1,31 @@
+package com.google.sps.data;
+
+import java.util.Comparator;
+
+/* Class that represent an event result in search. */
+public class EventResult {
+  private String eventId;
+  private float ranking;
```

Comment:
Why float? (purely for my understanding, since I kinda associate ranking with integers)

Reply:
So ranking is calculated using output from nlp library which is float value.